2-Node/nodeTool.js

```diff
@@ -0,0 +1,112 @@
+require("dotenv").config();
+const { Octokit } = require("@octokit/core");
+
+const regExp = new RegExp(process.argv[2]);
+
+const userName = "juan-fwn";
+const repositoryName = "apprenticeship";
+const token = process.env.GITHUB_API_KEY;
+
+const getGitHubAPI = async (URL) => {
+  const octokit = new Octokit({
+    auth: token,
+  });
+
+  const response = await octokit.request(URL);
+
+  return response;
+};
+
+const getCardsInfo = async (URL) => {
```

Comment:
I think in order for this function to be more clear it would be good to call it getCardInfo as it returns the data of a single card.